cmd/apps/confluentkafka.go

```diff
@@ -0,0 +1,140 @@
+		appOpts := types.DefaultInstallOptions()
+
+		wait, err := command.Flags().GetBool("wait")
+		if err != nil {
+			return err
+		}
+
+		kubeConfigPath, _ := command.Flags().GetString("kubeconfig")
+		namespace, _ := command.Flags().GetString("namespace")
+
+		updateRepo, err := command.Flags().GetBool("update-repo")
+		if err != nil {
+			return err
+		}
+		appOpts.WithHelmUpdateRepo(updateRepo)
+
+		overrides := map[string]string{}
+
+		enableZookeeper, err := command.Flags().GetBool("enable-zookeeper")
+		if err != nil {
+			return err
+		}
+		overrides["cp-zookeeper.enabled"] = strconv.FormatBool(enableZookeeper)
+
+		zookeeperServerCount, err := command.Flags().GetInt("zookeeper-server-count")
+		if err != nil {
+			return err
+		}
+		overrides["cp-zookeeper.servers"] = fmt.Sprintf("%d", zookeeperServerCount)
+
+		enableKafka, err := command.Flags().GetBool("enable-kafka")
+		if err != nil {
+			return err
+		}
+		overrides["cp-kafka.enabled"] = strconv.FormatBool(enableKafka)
+
+		kafkaBrokerCount, err := command.Flags().GetInt("kafka-broker-count")
+		if err != nil {
+			return err
+		}
+		overrides["cp-kafka.brokers"] = fmt.Sprintf("%d", kafkaBrokerCount)
+
+		enableSchemaRegistry, err := command.Flags().GetBool("enable-schema-registry")
+		if err != nil {
+			return err
+		}
+		overrides["cp-schema-registry.enabled"] = strconv.FormatBool(enableSchemaRegistry)
+
+		enableKafkaRest, err := command.Flags().GetBool("enable-kafka-rest")
+		if err != nil {
+			return err
+		}
+		overrides["cp-kafka-rest.enabled"] = strconv.FormatBool(enableKafkaRest)
+
+		enableKafkaConnect, err := command.Flags().GetBool("enable-kafka-connect")
+		if err != nil {
+			return err
+		}
+		overrides["cp-kafka-connect.enabled"] = strconv.FormatBool(enableKafkaConnect)
+
+		enableKSQLServer, err := command.Flags().GetBool("enable-ksql-server")
+		if err != nil {
+			return err
+		}
+		overrides["cp-ksql-server.enabled"] = strconv.FormatBool(enableKSQLServer)
+
+		enableControlCenter, err := command.Flags().GetBool("enable-control-center")
+		if err != nil {
+			return err
+		}
+		overrides["cp-control-center.enabled"] = strconv.FormatBool(enableControlCenter)
+
+		appOpts.
+			WithKubeconfigPath(kubeConfigPath).
+			WithOverrides(overrides).
+			WithValuesFile("values.yaml").
+			WithHelmURL("https://confluentinc.github.io/cp-helm-charts/").
+			WithHelmRepo("confluentinc/cp-helm-charts").
+			WithNamespace(namespace).
+			WithInstallNamespace(false).
+			WithWait(wait)
+
+		if _, err := apps.MakeInstallChart(appOpts); err != nil {
+			return err
+		}
+
+		fmt.Println(kafkaPostInstallMsg)
+
+		return nil
+	}
+
+	return kafka
+}
+
+const KafkaInfoMsg = ``
+
+const kafkaPostInstallMsg = `=======================================================================
+= Kafka has been installed.                                        =
+=======================================================================` +
```

Comment:
Let's give a link to the chart or some attempt at a post-install message that is helpful to the user?

Reply:
@developer-guy just this left I think?